MobileCenterAnalytics/MobileCenterAnalytics/Support/MobileCenterAnalytics.xcconfig

```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS = $(inherited) -framework Foundation -framework UIKit -framework CoreTelephony;
+OTHER_LDFLAGS = $(inherited) -framework Foundation -framework CoreTelephony;
```

Comment:
CoreTelephony is macOS 10.10+ only which okay in my opinion. We're supporting the latest 3 iOS versions, and we'd do the same on macOS with having 10.10 as our minimum OS version.